pages/jobs/ui-ux-dx-designer.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: 'UI/UX/DX Designer'
 date: 2020-12-02T16:56:52+01:00
-category: design
-closingOn: 01/15/2021
+category: Design
+closingOn: 02/15/2021
```

Comment:
@fmvilas what closing date should we set here?

Reply:
I don't know yet. Put something far in the future for now.

Reply:
@AceTheCreator set june please 😉

Reply:
Ok

Reply:
@derberg and @fmvilas I actually have a friend that fits in this role. Can I refer her?

Reply:
@AceTheCreator oh man, definitely! please share it with her and let her apply